### PR TITLE
Allow to include posts by ID (resolve #310)

### LINKED ISSUE #310: Listing posts/pages with post id?
Hi there - 
Many thanks for this great plugin and your very helpful documentation. Now, **is there a way to list posts with post ids**? Similar to the excludeposts function, only 'include'? Couldn't find the option anywhere - apologies if I should have!

### PR DESCRIPTION
**Pull request by hvianna**
This PR adds a `includeposts` parameter that allows to list selected posts/pages by ID. Also adds the corresponding option in the widget form.

Example usage:

[catlist post_type="page" includeposts="2,97" content="full"]

### CODE REVIEW

**Comment by picandocodigo** [comment]
Hi @hvianna, muito obrigado pelo Pull Request!

Sorry I haven't reviewed it yet. I recently came back from holidays and I'm still getting back to "work mode". I will review this and @zymeth25's Pull Requests as soon as possible and try to get the first update of 2019 out.

Cheers!

**Comment by hvianna** [comment]
No problem! I saw your "on vacation" status when I submitted it, so I knew it could take a little longer. Take your time!

File: include/lcp-widget.php [hunk 1]
```diff
@@ -58,6 +60,7 @@ function widget($args, $instance) {
       'excerpt_size' => $excerptsize,
       'exclude' => $exclude,
       'excludeposts' => $excludeposts,
+      'includeposts' => $includeposts,
```
Comment (zymeth25) [inline include/lcp-widget.php]: This is unnecessary. Actually this whole array here is kind of obsolete because of the merge I added on line 73

**Comment by picandocodigo** [comment]
@klemens-st can you test if you have permissions to merge this PR?

**Comment by klemens-st** [comment]
I can, also noticed mergo commits are disabled but that's fine.

**Pull request merged**